docs/cli/argo.md

```diff
@@ -107,7 +107,7 @@ argo [flags]
 * [argo cron](argo_cron.md)	 - manage cron workflows
 * [argo delete](argo_delete.md)	 - delete workflows
 * [argo get](argo_get.md)	 - display details about a workflow
-* [argo lint](argo_lint.md)	 - validate files or directories of workflow manifests
+* [argo lint](argo_lint.md)	 - validate files or directories of manifests
```

Comment:
I'm thinking if we should still even support `argo {template,cron,cluster-template} lint` if `argo lint` can now lint everything? Maybe "soft link" those commands to `argo lint` for compatibility?

Reply:
I thought making the other commands just call [this function](https://github.com/argoproj/argo-workflows/pull/5224/files#diff-c9b1f50c36281159db1f24f20050bb13512a9cea0749bdeb975c0ff8688712eaR76) with their relative kinds, but that would require me to move this function to `github.com/argoproj/argo-workflows/v3/cmd/argo/lint` because if I leave it where it is now it would cause a circular dependency.
Do you think I should do it?

Reply:
I think that would be good. @alexec thoughts?

Reply:
I agree-I think we should try and use the same code everywhere

Reply:
done